docs/conf.py

```diff
@@ -396,48 +385,35 @@ def find_and_replace(self, node):
     """
     if isinstance(node, nodes.reference) and 'refuri' in node:
       reference_uri = node['refuri']
-      if reference_uri.endswith('.md') and not reference_uri.startswith('http'):
-        reference_uri = reference_uri[:-3] + '.html'
-        node['refuri'] = reference_uri
-      else:
-        match = self.ANCHOR_REGEX.match(reference_uri)
-        if match:
-          node['refuri'] = '{0:s}.html#{1:s}'.format(
-              match.group('uri'), match.group('anchor'))
+      l2tdocs_prefix = 'https://github.com/log2timeline/l2tdocs/blob/'
+      if (
+          reference_uri.startswith(l2tdocs_prefix) and
+          not reference_uri.endswith('.asciidoc')):
+        node['refuri'] = reference_uri + '.md'
+        print('{0:s} -> {1:s}'.format(reference_uri, node['refuri']))
     return node
 
-  def traverse(self, node):
+  def _Traverse(self, node):
     """Traverse the document tree rooted at node.
 
     node(node): docutils node.
```

Comment:
add `Args:`

Reply:
Also use "[...]" => "node [node]: ..."

Reply:
Also node looks like a dict? or is it sphinx.node?

Reply:
Done.